### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Jac86** [nhn/toast-ui.doc#45]
## Summary

Given the current state of accessibility in web applications, we are required to provide an appropriate solution. The TUI Calendar is used in our web application. However, unfortunately, it does not meet the minimum accessibility requirements. 

The only question I found is 6 years old, and the response simply stated that this was not being planned at the time.

This raises the question: Are there further plans to make the TUI Calendar more accessible? If so, can this be expected to happen this year?

## Version

v1.15.3

## Additional context

Accessibility of the calendar to improve user-friendliness when navigating via keyboard and using screen readers.
